README.md

```diff
@@ -34,6 +34,8 @@ To install the cask (DMG/Application release build):
 
 ```
 brew install --cask gossip
+OR
+brew install --cask --no-quarantine nostorg/homebrew-nostr/gossip
```

Comment:
I had all of this extra stuff before but I think it is better to keep it concise as it is now.

The `--no-quarantine` flag is not needed here since we already remove the quarantine attributes during installation.

Reply:
Clearly something isn't happening...

![Screen Shot 2023-10-01 at 10 43 09 AM](https://github.com/nostorg/homebrew-nostr/assets/152159/4cf5f649-4ec6-4b5a-a012-3351eba2624e)

Reply:
> it ***may*** be necessary

Key word! I understand how it is confusing though.

Anyway, that is a caveat manually written into the cask in our repo here from before we managed to work around the issue mentioned. It's outdated now so maybe needs to be removed or updated. Thank you for the reminder.